twingate/internal/client/client.go

```diff
@@ -31,6 +31,8 @@ const (
 	headerAgent         = "User-Agent"
 	headerCorrelationID = "X-Correlation-Id"
 
+	ctxReqIDKey = "Ctx_req_id_key"
```

Comment:
lets give this a decent name - `X-Twingate-Request-Id`

Reply:
yes, updated it